rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -92,7 +92,7 @@ def _is_synchronized(current_block: int, latest_block: int) -> tuple[bool, str]:
             - message: A message containing information on what the status is.
     """
     message = ''
-    if current_block < (latest_block - 20):
+    if current_block < (latest_block - 20) or current_block > (latest_block + 20):
```

Comment:
Don't do this since for example arbitrum publishes a lot of blocks per second

```suggestion
    if current_block < (latest_block - 20)
```

if it's after the latest block it's fine